### PR TITLE
update: added information

### PR DESCRIPTION
**Pull request by Danielecina**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

File: docs/console/company-configuration/clusters-management/cluster-setup.mdx [hunk 1]
```diff
@@ -158,6 +159,31 @@ You can choose between the automatic procedure and the manual one.
         - {{NAME_OF_THE_SERVICE_ACCOUNT_USED_FOR_THE_DEPLOY}}
   ```
 
+  #### Monitoring custom resources from the Console
+
+  Following these guidelines for configuring custom resources. 
+  We need to make sure that the console has the correct permissions 
+  to display the information to users.
```
Comment (malta895) [inline docs/console/company-configuration/clusters-management/cluster-setup.mdx:166]:
```suggestion
  You need to make sure that the Console has the correct permissions 
  to display the information to the users.
```

File: docs/console/company-configuration/clusters-management/cluster-setup.mdx [hunk 2]
```diff
@@ -158,6 +159,31 @@ You can choose between the automatic procedure and the manual one.
...
+      verbs:
+        - get
+        - list
+  ```
+
+  After adding the necessary permissions follow this [guide](/console/design-your-projects/custom-resources/custom-resources.md) to see your custom resources inside the console
+
   ### ClusterRoleBindings
 
   Finally, we **must** associate the roles with the service account using the `ClusterRoleBindings` that can be created using the following template:
```
Comment (malta895) [inline docs/console/company-configuration/clusters-management/cluster-setup.mdx:189]:
```suggestion
  Finally, make sure to associate the roles with the service account using the `ClusterRoleBindings` that can be created using the following template:
```

File: docs/console/company-configuration/clusters-management/cluster-setup.mdx [hunk 3]
```diff
@@ -36,7 +36,8 @@ You can choose between the automatic procedure and the manual one.
 
   ### ClusterRole
 
-  Next, we must define the roles needed by the previously created service account. These are the minimal roles required by the Console to work.
+  Next, we must define the roles needed by the previously created service account.
```
Comment (malta895) [inline docs/console/company-configuration/clusters-management/cluster-setup.mdx:39]:
```suggestion
  Next, make sure to define the roles needed by the previously created service account. 
```

File: docs/console/company-configuration/clusters-management/cluster-setup.mdx (hunk 2 above)
Comment (malta895) [reply docs/console/company-configuration/clusters-management/cluster-setup.mdx:189]: must feels like too much on a documentation, plus there is inconsistency between 'we' and 'you'. I'd use always 'you', this is a guide for the user

**Review by mattia-fumo — APPROVED** [review]

**Review by malta895 — APPROVED** [review]

**Pull request merged**